packages/lib/services/synchronizer/syncInfoUtils.ts

```diff
@@ -82,7 +83,7 @@ export async function uploadSyncInfo(api: FileApi, syncInfo: SyncInfo) {
 	await api.put('info.json', syncInfo.serialize());
 }
 
-export async function fetchSyncInfo(api: FileApi): Promise<SyncInfo> {
+export async function fetchSyncInfo(api: FileApi, checkSyncedItems = false): Promise<SyncInfo> {
```

Comment:
Consider defaulting to `true` or removing the parameter.

```suggestion
export async function fetchSyncInfo(api: FileApi, checkSyncedItems = true): Promise<SyncInfo> {
```

Currently, `fetchSyncInfo` is used in:
- Tests
- `Synchronizer.ts`
- `MigrationHandler`

- In the first use of `fetchSyncInfo` in `Synchronizer.ts`, `checkSyncedItems` is already manually set to `true`.
- It should also be fine for `checkSyncedItems` to be `true` in the other use of `fetchSyncInfo`.
- In `MigrationHandler.checkCanSync`, it may make more sense for `checkSyncedItems` to be `true`. Although `.checkCanSync` is currently only used after setting up the sync target, I don't think it should fail if the sync target is empty and `remoteInfo` is not provided.

Reply:
I think you're right, that the parameter can be removed and always passed as true for this method. I'll remove it and retest. The other comments make sense as well